src/components/pages/PageBase.js

```diff
@@ -26,12 +28,19 @@ function PageBase({ children, ...props }) {
     history.replace('/');
   };
 
+  useEffect(() => {
+    if (localStorage.getItem(USER)) {
+      Axios.get('/users/').then(({ data }) => setAdminUrl(data?.results[0].admin_url || ''));
```

Comment:
I'm a little worried about this line since I'm not familiar with how the data gets formed. Will there ever be a case where `data` is returned with an empty `results` array? Will `admin_url` always be present in every `result`?

Reply:
> Will there ever be a case where data is returned with an empty results array?

Results should always contain a single item list.

> Will admin_url always be present in every result?

Yup